3_javascript/javascript1/week1/script.js

```diff
@@ -0,0 +1,86 @@
+//Age-ify (A future age calculator)
+
+const yearOfBirth = 1995;
+const yearFuture = 2045;
+console.log(typeof yearFuture);
+const curentYear = 2023;
+const age = curentYear - yearOfBirth;
+console.log(age);
+const futureAge = yearFuture - curentYear + age;
+console.log(futureAge);
+console.log(`You will be ${futureAge} years old in ${yearFuture}`);
+
+//Goodboy-Oldboy (A dog age calculator)
+
+const dogYearOfBirth = 2018;
+const dogYearFuture = 2045;
+const dogYear = 2023;
+const result = dogYear - dogYearOfBirth;
+console.log(result);
+const dogFutureAge = dogYearFuture - dogYear + result;
+console.log(dogFutureAge);
+const shouldShowResultInDogYears = true;
+if (shouldShowResultInDogYears === true) {
+  console.log(`Your dog will be ${dogFutureAge} dog years old in ${dogYearFuture}`);
+} else {
+  console.log(`Your dog will be ${dogFutureAge} human years old in ${dogYearFuture}`);
+}
+
+//Housey pricey (A house price estimator)
+
+const volumeInMeters1 = 8 * 10 * 10;
+const housePrice1 = 2500000;
```

Comment:
I would suggest escaping generic variable names and instead choosing names that tell what the variable is used for. That will make the code more readable. So instead of housePrice1 and housePrice2, I would suggestusinge thePricOfTheHouseJuliaWants and thePriceOfTheHousePeterWants.